pkg/util/controllerfinder/pods_finder.go

```diff
@@ -120,6 +121,7 @@ func (r *ControllerFinder) GetPodsForRef(apiVersion, kind, ns, name string, acti
 			FieldSelector: fields.SelectorFromSet(fields.Set{fieldindex.IndexNameForOwnerRefUID: string(uid)}),
 		}
 		pods, err := listPods(&listOption)
+		klog.V(5).InfoS("result of list pods with owner ref uid", "pods", len(pods), "err", err, "refUid", uid)
```

Comment:
The uid is not very suitable for viewing logs, please update it to be more log-friendly with some key.

Reply:
it is very useful to check whether the pods are listed properly while debugging